Libraries/LibGUI/Widget.cpp

```diff
@@ -953,14 +953,22 @@ void Widget::set_override_cursor(Gfx::StandardCursor cursor)
 }
 
 bool Widget::load_from_gml(const StringView& gml_string)
+{
+    return load_from_gml(gml_string, [](const String& class_name) -> RefPtr<Widget> {
+        dbg() << "Class '" << class_name << "' not registered";
```

Comment:
pls2use `dbgln()` in new code

Reply:
but but but it's old code just in a new location

Reply:
done